package.json

```diff
@@ -92,7 +94,7 @@
     "react-native-web": "~0.12.0",
     "react-native-windows": "^0.63.41",
     "react-test-renderer": "16.13.1",
-    "semantic-release": "^17.2.1"
+    "semantic-release": "^17.4.6"
```

Comment:
Should we bump to 18?

Reply:
I did not bump to latest due to node version required (14), while our CI images are at 12

Reply:
Should we bump Node to 14 (in a separate PR)?

Reply:
Yes, that'd require more compatibility checks on CI, so ideally separate PR